specification/healthdataaiservices/HealthDataAIServices.Deidentification/models.tsp

```diff
@@ -46,6 +46,14 @@ model Job {
   @visibility("read")
   createdAt: utcDateTime;
 
+  @doc("Error code for complete job failures.")
+  @visibility("read")
+  errorCode?: string;
+
+  @doc("Error message for complete job failures.")
```

Comment:
Is it only available when job fails? 

There is a recommended error type as described here https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#handling-errors, do you want to follow the same data structure?

Reply:
Ahhhh great point, I figured that could only be used for error responses. I've updated the PR to use this for jobStatuses as well.

Also Yes, this field will only be available when the job has Failed.